djangocms_rest_api/serializers/base.py

```diff
@@ -239,12 +239,14 @@ def _get_declared_fields(attrs):
     return serializer_class
 
 
-def get_serializer_class(plugin=None, model=None):
+def get_serializer_class(plugin=None, model=None, instance=None):
     serializer_class = None
     if plugin:
+        # import pdb
+        # pdb.set_trace()
         serializer_class = getattr(plugin, 'serializer_class', None)
-        if not serializer_class:
-            serializer_class = plugin_serializer_mapping.get(plugin.plugin_type)
+        if not serializer_class and instance:
```

Comment:
plugin does not have attribute plugin_type or at least some like LinkPlugin. Got AttributeError in there